lib/pry/testable/evalable.rb

```diff
@@ -8,6 +8,21 @@ def pry_tester(*args, &block)
     end
   end
 
+  #
+  # For super-high-level integration testing.
```

Comment:
When I read docs I start like this "The pry_repl_tester method ...". When I read this I see "The pry_repl_tester method is for super-high-level integration testing". Can we describe what this method does or remove this sentence?